tests/index.js

```diff
@@ -106,6 +106,8 @@ describe('Thorn', () => {
                         name: 'TestRecord1',
                         testField1: 'TestField1data',
                         testField2: 'TestField2data',
+                        testField3: false,
```

Comment:
I was on the fence between adding a new test-case such as `it('should fill required fields when respective values aren\'t supplied' or making use of one of the already existing ones, went with the latter 'cause it made me feel that having a separate test-case for something that is kind of being tested already was overkill. I'm open to suggestions though.